### PR TITLE
Fix workflow to update server names

### PR DESCRIPTION
**Pull request by flodolo**
Workflow is currently broken (missing deps), and has the same name of another workflow.

### CODE REVIEW

**Comment by flodolo** [comment]
Confirmed that it's working (manually triggered from branch, created #9519)

File: .github/workflows/i18n_server_list_update.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: "true"
+      - uses: actions/setup-python@v5
```
Comment (flodolo) [inline .github/workflows/i18n_server_list_update.yaml:24]: Using v5 directly, as there's another PR trying to update all the workflows #9520

**Review by brizental — APPROVED** [review]

**Comment by brizental** [comment]
I guess we should drop 96bbdec138ba7f483be8807e5fd87ca4423a8165 before merge.

**Comment by flodolo** [comment]
> .



> I guess we should drop [96bbdec](https://github.com/mozilla-mobile/mozilla-vpn-client/commit/96bbdec138ba7f483be8807e5fd87ca4423a8165) before merge.

Yes, I can manually trigger the workflow after this is merged. I originally didn't realize the PR was opened against the branch.

**Pull request merged**